src/Mod/Part/App/PrimitiveFeature.cpp

```diff
@@ -940,12 +942,38 @@ App::DocumentObjectExecReturn *Spiral::execute(void)
         gp_Pnt2d cend(u, v);
         end = cend;
 
-        Handle(Geom2d_TrimmedCurve) segm = GCE2d_MakeSegment(beg , end);
+//         Handle(Geom2d_TrimmedCurve) segm = GCE2d_MakeSegment(beg , end);
+// 
+//         TopoDS_Edge edgeOnSurf = BRepBuilderAPI_MakeEdge(segm , surf);
+//         TopoDS_Wire wire = BRepBuilderAPI_MakeWire(edgeOnSurf);
+//         BRepLib::BuildCurves3d(wire);

```

Comment:
Rather than commenting this out, it can (should?) be deleted altogether. There will be a history of this old implementation available in git.